### PR TITLE
Feature/Find all GMP usages

### PR DESCRIPTION
**Pull request by bombifighter**
Added the initial files for the repository.

Also added the first version of the GMP function detector script, commited as an executable.

Please review the changes!

Thanks. :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+This repository's aim to contain a reusable script, to detect all GMP function usages in different repositories.
+
+In order to do that we create a simple script, which lists all the functions in a specified folder.
+
+## Usage
+
+You may need to specify the path to the folder in the first prameter, use the following command:
+
+~~~~bash
+./mpz_search.sh [path]
```
Comment (battila7) [inline README.md]: For me, it's not evident whether the `path` argument is required or not, and the behavior when it's missing. What do you think, can we make this usage description a bit more exact?

File: mpz_search.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+grep "mpz_[a-z_0-9]*" $1 -a -r -o -h --exclude "mpz_search.sh" | sort -d | uniq
```
Comment (battila7) [inline mpz_search.sh]: Functionality-wise, it looks great! 👍 I've only noticed one small thing: currently it searches through all files, which seems unnecessary. Is it possible to grep in only `.c`˙and `.h` files?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Gmp-detector
+
+This repository's aim to contain a reusable script, to detect all GMP function usages in different repositories.
+
+In order to do that we create a simple script, which lists all the functions in a specified folder.
```
Comment (battila7) [inline README.md]: The content is pretty good, however, there are some smaller grammatical issues. Can you check it with grammarly, maybe?

**Review by battila7 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bombifighter) [reply README.md]: Sure, I will describe it more exact. :smile:

File: README.md (hunk 3 above)
Comment (bombifighter) [reply README.md]: Sure!

File: README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Gmp-detector
+
+This repository's aim to contain a reusable script, to detect all GMP function usages in different repositories.
```
Comment (AVecsi) [inline README.md]: The README says that the aim is to find "all" the GMP functions, however, the script you made is not a general-purpose function. It's okay if the repository has that nice aim, but can you make some additional description to make it clear that the current script can not detect all the GMP functions?

**Review by AVecsi — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (bombifighter) [reply README.md]: Of course, thank you for your remark.

File: mpz_search.sh (hunk 2 above)
Comment (bombifighter) [reply mpz_search.sh]: Thanks!
I'll look for a solution.

**Comment by bombifighter** [comment]
I made the requested changes.

What do you think? :smile:

**Review by battila7 — APPROVED** [review]
Nice work, ready to merge in my opinion.

**Review by AVecsi — APPROVED** [review]
Seems good, approved.

**Pull request merged**